pyproject.toml

```diff
@@ -31,7 +31,7 @@ classifiers = [
     "Topic :: Software Development :: Libraries",
 ]
 dependencies = [
-    "mirascope>=1.2.0",
+    "mirascope>=1.5.2",
```

Comment:
nit: i believe this will now need `>=1.6.0` because of another bug fix in that release for using built-in response models

Reply:
changed to 1.6.1